securedrop/journalist_app/admin.py

```diff
@@ -334,10 +334,8 @@ def new_password(user_id: int) -> werkzeug.Response:
             abort(404)
 
         password = request.form.get('password')
+        logout_user(user_id)
         if set_diceware_password(user, password) is not False:
-            if user.last_token is not None:
-                revoke_token(user, user.last_token)
```

Comment:
This was probably a bug since it was calling the JWT revocation function on journalists OTP, probably due to the common usage of 'token'.